configs/default/HBRO-KAKUTEF7.config

```diff
@@ -1,7 +1,7 @@
 # Betaflight / STM32F745 (S745) 4.0.0 Apr  3 2019 / 14:32:23 (22b9f3453) MSP API: 1.41
 
 #mcu STM32F745
-#wiki https://betaflight.com/docs/boards/KAKUTEF7
+#wiki https://betaflight.com/docs/wiki/boards/KAKUTEF7
```

Comment:
Without this line we also reach board documentation. This can be removed like all other #wiki directives.

Reply:
Hmm, something wnet wrong in my PR :-(
Just keep those for reference :-)